server/routes/induction/update/wantToAddQualificationsFormValidator.test.ts

```diff
@@ -11,7 +11,7 @@ describe('wantToAddQualificationsFormValidator', () => {
       const expected: Array<Record<string, string>> = []
 
       // When
-      const actual = validateWorkedBeforeForm({ wantToAddQualifications: YesNoValue.YES }, prisonerSummary)
```

Comment:
Looks like this import was named incorrectly before.

Reply:
Good spot 👍